core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+
+{% block maintitle %}
+  Oppia Editor
+{% endblock maintitle %}
+
+{% block subtitle %}
+  {{ title }}
+{% endblock subtitle %}
+
+
+{% block header_js %}
+  {{ super() }}
+  <script type="text/javascript">
+    GLOBALS.can_edit = JSON.parse('{{can_edit|js_string}}');
+  </script>
+
+  <style>
+    html, body {
+      background-color: #eee;
+    }
+  </style>
+
+  {{dependencies_html}}
+{% endblock header_js %}
+
+
+{% block content %}
+  <div ng-controller="CollectionEditor">
+	  <p>title: <[collection.title]></p>
```

Comment:
FYI, I think you need to enclose html tags in backquotes (`) to ensure they show up in github comments.

Also, I think you're using tabs here and not spaces for the indentation, that's why it looks different.